sources/packages/web/src/components/aest/students/EditCASSupplierInfoModal.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <v-form ref="addCASSupplierForm">
+    <modal-dialog-base
+      title="Update supplier/Site information"
+      :showDialog="showDialog"
+    >
+      <template #content>
+        <error-summary :errors="addCASSupplierForm.errors" />
+        <div class="pb-5">
+          <span class="label-value"
```

Comment:
For accessibility please use `<p class="label-value">`. Some modals already follow it.